docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md

```diff
@@ -8,6 +8,18 @@ This is the primary flow and purpose for RAD Lab UI. A step-by-step graphical in
 
 Admins and Users are both able to deploy modules.
 
+Below video provides a quick overview on how to setup RAD Lab UI and deploy RAD Lab modules with the same.
+
+<iframe
+    width="980"
+    height="480"
+    src="https://www.youtube.com/embed/Ir79gCzleLU"
+    frameborder="0"
+    allow="autoplay; encrypted-media"
```

Comment:
Will this autoplay? I don't think that's desireable.

Reply:
Removed autoplay